src/Tempest/Database/Drivers/PostgreSqlDriver.php

```diff
@@ -0,0 +1,40 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Tempest\Database\Drivers;
+
+use SensitiveParameter;
+use Tempest\Database\DatabaseDriver;
+
+final class PostgreSqlDriver implements DatabaseDriver
+{
+    public function __construct(
+        #[SensitiveParameter]
+        public string $host = 'localhost',
+        #[SensitiveParameter]
+        public string $port = '5432',
+        #[SensitiveParameter]
+        public string $username = 'postgres',
+        #[SensitiveParameter]
+        public string $password = '',
+        #[SensitiveParameter]
+        public string $database = 'app',
+    ) {
```

Comment:
maybe it's because I'm working with Laravel for a long time. I always use an empty comment here. 🤷

Reply:
Yeah not needed here :)